src/main/java/org/terasology/commonworld/geom/CircleUtility.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.commonworld.geom;
+
+import org.terasology.joml.geom.Circlef;
+import org.terasology.joml.geom.Rectanglef;
+
+public class CircleUtility {
+    public static boolean intersect(Circlef cir, Rectanglef rect) {
```

Comment:
Should this move to `joml-geom` at a later point?

Reply:
Probably, just wanted to get everything working.